controller/iteration_test.go

```diff
@@ -422,8 +462,16 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 	var rootIterationObj iteration.Iteration
 	var otherIterationObj iteration.Iteration
 	application.Transactional(db, func(app application.Application) error {
+		owner := &account.Identity{
+			ID:           uuid.NewV4(),
+			Username:     "new-space-owner-identity",
+			ProviderType: account.KeycloakIDP,
+		}
+		errCreateOwner := app.Identities().Create(context.Background(), owner)
+		require.Nil(t, errCreateOwner)
 		spaceObj = space.Space{
-			Name: "Test 1-" + uuid.NewV4().String(),
+			Name:    "Test 1-" + uuid.NewV4().String(),
```

Comment:
I'll use our function `name := testsupport.CreateRandomValidTestName("TestSuccessCreateSpaceAndRootAreaAndIterations-")`. Remember we need to be able to track the created items with a valid name. This function creates a space name that satisfies our main restrictions.